src/schema/objects/columns.yaml

```diff
@@ -47,6 +47,45 @@ age:
     "Units": "year",
     "Maximum": 89,
   }
+AP_angle:
```

Comment:
Need to add `anatomical_reference_point` for when you want to use something other than begma